### DISCUSSION TITLE
Adding rich text node edition

### DISCUSSION THREAD

**Discussion by delcroip** [jgraph/drawio#4273] [General]
* [ x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.
(not the same as #711 )

**Is your feature request related to a problem? Please describe.**
I am using drawio to create decision tree (xml parsed with python) where the node content may be display in a questionnaire, being able to have a richtext editor to update the node content would enable to have nicer looking content in the nodes

**Describe the solution you'd like**
Either;

   * clicking on the contextual menu "Edit Rich text" (a bit like edit link) that would show a modal box with rich text editor
   *  on the panel, having a rich text tab

**Describe alternatives you've considered**
injecting data from HTML files but really not user friendly

**Additional context**

I seems that there is richtext js lib available such as https://quilljs.com/playground/ , would be great if one could be compatible

**Comment by delcroip** [comment]
Thanks you for the label, is there anyway to do it via a plugin ? (I could check to get devs if it is possible to do it)

**Comment by image72** [comment]
text `text;html=1` satiable ？

**Comment by dzylikecode** [comment]
@delcroip I developed a plugin just now. Would you like to try ? https://github.com/dzylikecode/drawio-plugin-markdown

feel free to come up with any suggestion :smile:
